src/client.ts

```diff
@@ -416,30 +417,43 @@ export class ClobClient {
         return this.get(`${this.host}${endpoint}`, { headers, params });
     }
 
-    public async createOrder(userOrder: UserOrder, tickSize?: TickSize): Promise<SignedOrder> {
+    public async createOrder(
+        userOrder: UserOrder,
+        options?: Partial<CreateOrderOptions>,
+    ): Promise<SignedOrder> {
         this.canL1Auth();
 
         const { tokenID } = userOrder;
-        tickSize = await this._resolveTickSize(tokenID, tickSize);
 
-        return this.orderBuilder.buildOrder(userOrder, tickSize);
+        const tickSize = await this._resolveTickSize(tokenID, options?.tickSize);
+        const negRisk = options?.negRisk ?? false;
```

Comment:
As the `negRisk` flag is a param of this function, the frontend (or any other client) has to know in advance if the market is neg risk or not, is that information available in this context or should we add a new CLOB request to resolve that?

cc: @mttwlkr @L-Kov

Reply:
Yes, the frontend will known at the time of calling whether the market is neg-risk or not.  We considered adding an extra call to the CLOB to determine locally (to this package) whether the market is neg-risk but opted to make these calls as light-weight as possible.

Reply:
it makes sense, thanks!